src/main/java/org/opensearch/security/transport/SecurityRequestHandler.java

```diff
@@ -338,7 +338,7 @@ protected void addAdditionalContextValues(final String action, final TransportRe
         String extensionUniqueId = getThreadContext().getHeader("extension_unique_id");
         if (extensionUniqueId != null) {
             ExtensionsManager extManager = OpenSearchSecurityPlugin.GuiceHolder.getExtensionsManager();
-            if (extManager.lookupExtensionSettingsById(extensionUniqueId).isPresent()) {
+            if (extManager.lookupInitializedExtensionById(extensionUniqueId).isPresent()) {
```

Comment:
this change should be reverted as lookupExtensionSettingsById() is the correct method call.